test/Selenium.WebDriver.IEDriver.NuPkg.Test.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Upgrade to .NET 6.0?

Reply:
@SIkebe Thank you for the proposal!

I also upgraded that project.

https://github.com/jsakamoto/nupkg-selenium-webdriver-iedriver/commit/2baab4caf1a05f2bc9ec66a7e1391f4feb6a4602?diff=split#diff-fffdd83d7da541666a2f380958202aa2c8dc4dfdd17ad06453b4ef1279c76d4cR4